### DISCUSSION TITLE
About wifi connection in ipxe.

### DISCUSSION THREAD

**Discussion by hw0505** [ipxe/ipxe#603] [Q&A]
I want to connect to wireless network and have ssid and key configured in ipxe shell, but the connection fails. Its error message is "802.11 0x2b7b2810 probe: found nothing for '3529' ('3529' is a ssid)".

Does anyone know how to solve this problem?

Thank you very much !

**Comment by NiKiZe** [comment]
What is the full error message, including the ipxe.org URL in the message?

**Reply by hw0505** [reply]
All error messages are shown below：

Waiting for link-up on net0 ........802.11 0x2b7b2810 probe: found nothing for '3529'
802.11 0x2b7b2810 association failed (state=0000): Connection timed out (https://ipxe.org/4c1f6092)

**Reply by hw0505** [reply]
I still get the same error when I use wifi without password.
please!help!

**Reply by NiKiZe** [reply]
On that page you have DEBUG=.... So the steps are there. Please post output from setting ssid to when it times out. Which should have several 80211 debug outputs.
